spec/paths/2fa/2fa-tokens@acions@verify.ts

```diff
@@ -0,0 +1,40 @@
+import { OperationObject, PathItemObject, ResponseObject } from 'openapi3-ts';
+import { ref as toCodeRef } from '../../components/schemas/2fa/token/requests/recipient';
+import { ref as errorResponseRef } from '../../components/responses/error';
+import { ref as tokenResponseRef } from '../../components/schemas/2fa/token/responses/token';
+
+const post: OperationObject = {
+  description: 'Verify a token',
```

Comment:
**VERIFICAR**

Não seria **Send the code to the user**?

Como tem um recurso da API chamado "Resend the code to the user", se o objetivo é enviar o código pro usuário mesmo, o jeito que está escrito pode criar confusão

**+**
Se é realmente enviar o código pro usuário, tem que criar um novo schema pra esse recurso (recurso no sentido desse recurso CRUD específico da API).
Seria substiuir o `toCodeRef` (na L14 abaixo) por algo novo, pois a propriedade `code` desse recurso seria **code sent to the user** ao invés de **code sent by the user**
(https://github.com/zenvia/zenvia-openapi-spec/pull/237/files#diff-2874fdb3c8bbb35a7b0fd01a7356a853c4370ccd0e393e57c760a0de8892b938R17)